protocol/protos/NetRemoteDataStream.proto

```diff
@@ -23,8 +23,54 @@ message DataStreamUploadData
     bytes Data = 1;
 }
 
+message DataStreamDownloadData
+{
+    bytes Data = 1;
+    uint32 SequenceNumber = 2;
+    DataStreamOperationStatus Status = 3;
```

Comment:
Is `Data` only present/valid when `Status == Succeeded`? If so, suggest re-ordering these such that always-present/valid fields are first, then optional/conditional ones following.

Reply:
Yes that's true. I can re-order the fields so that `Status` is first since that will always be present.